src/backend/joanie/core/models/certifications.py

```diff
@@ -155,9 +150,8 @@ def get_document_context(self, language_code=None):
 
     def save(self, *args, **kwargs):
         """On creation, create a context for each active languages"""
-        is_new = self.pk is None
-
+        is_new = self.created_on is None
         if is_new:
             self._set_localized_context()
 
-        super().save(*args, **kwargs)
+        models.Model.save(self, *args, **kwargs)
```

Comment:
What is the difference ?

Reply:
the difference is the first parent is not called, without it multiple queries get executed and it's not needed and tests dont pass